src/test/java/org/assertj/core/error/ShouldContain_create_Test.java

```diff
@@ -48,11 +48,11 @@ public void should_create_error_message() {
     String message = factory.create(new TextDescription("Test"));
     // THEN
     then(message).isEqualTo(format("[Test] %n" +
-                                   "Expecting:%n" +
+                                   "Expecting ArrayList:%n" +
                                    " <[\"Yoda\"]>%n" +
                                    "to contain:%n" +
                                    " <[\"Luke\", \"Yoda\"]>%n" +
-                                   "but could not find:%n" +
+                                   "but could not find the following elements:%n" +
```

Comment:
add tests in this class to covert the different kind of GroupTypeDescription (map, arrays, ...)